carmajava/rsumetering/src/main/java/gov/dot/fhwa/saxton/carma/rsumetering/HoldingState.java

```diff
@@ -92,6 +99,15 @@ public void onMobilityOperationMessage(MobilityOperation msg) {
     double mergeDist = Double.parseDouble(params.get(1));
     double speed = Double.parseDouble(params.get(2));
 
+    //perform sanity checks on the received values
+    if (meterDist < MIN_METER_DIST  ||  meterDist > MAX_METER_DIST  ||
+        mergeDist < MIN_MERGE_DIST  ||  mergeDist > MAX_MERGE_DIST  ||
+        speed < MIN_SPEED           ||  speed > MAX_SPEED) {
+      log.warn("Received operation message with suspect strategy values. meterDist = " + meterDist +
+                ", mergeDist = " + mergeDist + ", speed = " + speed);
+      return;
```

Comment:
The RSU is holding the vehicle still in this state so we should send a NACK message here.